File: api/app/Models/Registers/NCM.php
<?php

namespace App\Models\Registers;

use Illuminate\Database\Eloquent\Model;

class NCM extends Model
{
    protected $table = 'ncms';

    protected $fillable = [
        'issuer_id',
        'ncm',
        'description',
        'nacional_federal',
        'importados_federal',
        'estadual',
        'municipal',
        'inicio',
        'fim',
        'chave',
        'versao',
    ];
}

File: Services/NCMService/pkg/schemas/ncm.go
package schemas

import "time"

type NCM struct {
	Ncm                string
	Description        string
	Nacional_federal   float64
	Importados_federal float64
	Estadual           float64
	Municipal          float64
	Inicio             time.Time
	Fim                time.Time
	Chave              string
	Versao             string
}
